src/code/z_actor.c

```diff
@@ -3003,8 +3006,12 @@ Actor* Actor_Find(ActorContext* actorCtx, s32 actorId, s32 actorCategory) {
     return NULL;
 }
 
-void func_80032C7C(GlobalContext* globalCtx, Actor* actor) {
-    globalCtx->actorCtx.unk_00 = 5;
+/**
+ * Play the death sound effect and flash the screen white for 4 frames.
+ * While the screen flashes, the game freezes.
+ */
+void Actor_PlayDeathFx(GlobalContext* globalCtx, Actor* actor) {
```

Comment:
I feel like the name or documentation for this function should include "enemy" somewhere so it doesn't get confused with the player death. Also I think this would typically be called a "finishing blow" effect, but I don't know if it's worth including that in some of the names or documentation.

Reply:
I like finishing blow, yeah.
`Actor_EnemyPlayFinishingBlowFx` is a bit long, but its not like youll see this function a ton.
My instinct almost wants to make the system `Enemy_` but im not sure how many enemy specific functions we'll have lol